gen3-cli/cot/backend/test/templates/cfn_structure_obj_block.py

```diff
@@ -0,0 +1,36 @@
+def CFNStructure(JSONValidator):
```

Comment:
why it's done this way, with class created each time instead of just being defined once and only instances created?

Reply:
1. It's true that I can use unwrapped classes inside a template. 
2. But I can't use imports inside a template block because testcase pytest is a single file.
3. All validators have a base class that I can't import inside template block file, therefore to keep them as valid python and template block files I use function wrapped classes to build them at runtime and perform tests of the checks.

Reply:
Could you please comment it in the code (shortly) for the future generations; because somebody may be unaware of it and try to fix the code by removing it (in some years from now) and getting some strange errors.

Reply:
Sure, no problem.